ouster-ros/src/os_sensor_node.cpp

```diff
@@ -529,11 +530,13 @@ sensor::sensor_config OusterSensor::parse_config_from_ros_parameters() {
         }
     }
 
+    retry_configuration = get_parameter("retry_configuration").as_bool();
+
     return config;
 }
 
 sensor::sensor_config OusterSensor::parse_config_from_staged_config_string() {
-    sensor::sensor_config config = sensor::parse_config(staged_config);
+    config = sensor::parse_config(staged_config);
```

Comment:
Yep, changed the logic so that now this and the other methods will still srill return the config, that will be set as the member variable. And the other methods that use the config just use the member variable and don't have it as argument anymre